tests/lib/test_github_validators.py

```diff
@@ -3,9 +3,28 @@
 from jobrunner.lib.github_validators import (
     GithubValidationError,
     validate_branch_and_commit,
+    validate_repo_url,
 )
 
 
+def test_validate_repo_url():
+    validate_repo_url(
+        "https://github.com/opensafely-core/test-public-repository.git",
+        ["opensafely-core"],
+    )
+
+
+def test_validate_repo_url_reject():
+    with pytest.raises(
+        GithubValidationError,
+        match="must belong to one of the following Github organisations",
+    ):
+        validate_repo_url(
+            "https://github.com/blackhat/test-private-repository.git",
+            ["opensafely-core"],
+        )
```

Comment:
there was apparently no test coverage of this validation failure, so although it was working correctly I thought it prudent to write a test!